From 093d5ca53325186d70486b0361c1626b2d2fb10c Mon Sep 17 00:00:00 2001
From: Herman van Hazendonk <github.com@herrie.org>
Date: Tue, 8 Feb 2022 23:20:43 +0100
Subject: [PATCH] security_policy.conf: Add paths for LuneOS

So that apps actually render something and don't show network connection error.
Add the following paths:

For system apps:
/usr/palm/applications

For user installed apps:
/media/cryptofs/apps/usr/palm/applications/

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Inappropriate [LuneOS specific]

 files/launch/security_policy.conf | 4 ++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/files/launch/security_policy.conf b/files/launch/security_policy.conf
index cc29f96..6251de3 100644
--- a/files/launch/security_policy.conf
+++ b/files/launch/security_policy.conf
@@ -8,7 +8,9 @@
 7/path=/var/palm/license/
 8/path=/usr/palm/license/
 9/path=/usr/palm/plugins/license/
-size=9
+10/path=/usr/palm/applications/
+11/path=/media/cryptofs/apps/usr/palm/applications/
+size=11
 
 [Trusted]
 1/path=/media/
